### PR TITLE
Beautify Landing page

### PR DESCRIPTION
**Pull request by Lighteros**

### CODE REVIEW

**Comment by Lighteros** [comment]
Added slider for image show, 2 sections for introduction.

File: package.json [hunk 1]
```diff
@@ -28,7 +28,11 @@
 		"@cosmos-kit/ledger": "^2.8.0",
 		"@cosmos-kit/react": "2.12.0",
 		"@cosmos-kit/station": "^2.7.0",
+<<<<<<< HEAD
+		"@headlessui/react": "^2.0.3",
+=======
```
Comment (ismellike) [inline package.json]: want both here

File: src/config/categories.development.json [hunk 2]
```diff
@@ -73,18 +73,6 @@
 					"title": "Dota 2",
 					"img": "/category_images/dota-2.jpg",
 					"category_id": 17
-				},
-				{
-					"url": "nba-2k",
```
Comment (ismellike) [inline src/config/categories.development.json:78]: don't want these changes deleted

File: app/components/competition/view/ViewCompetition.tsx [hunk 3]
```diff
@@ -187,16 +187,14 @@ const ViewCompetition = ({
 						host={competition.host}
 					/>
 				)}
-				{competition.is_expired &&
-					status !== "inactive" &&
-					status !== "pending" && (
```
Comment (ismellike) [inline app/components/competition/view/ViewCompetition.tsx]: lost status !== 'pending' check here

File: app/components/NavBar.tsx [hunk 4]
```diff
@@ -184,18 +215,120 @@ export default function AppNavbar() {
...
+								leaveFrom="opacity-100 translate-y-0"
+								leaveTo="opacity-0 translate-y-1"
+							>
+								<PopoverPanel
+								anchor="bottom"
+								className="divide-y divide-white/5 mt-5 backdrop-blur-lg backdrop-saturate-150 bg-background/70 rounded-xl text-sm/6 [--anchor-gap:var(--spacing-5)]"
+								>
+								<div className="p-3">
+									<a className="block rounded-lg py-2 px-3 transition hover:bg-[#FF8000]" href="/dao/dao">
+										<p className="font-semibold text-white">Twitter</p>
```
Comment (ismellike) [inline app/components/NavBar.tsx]: just get the links updated here :)

File: package.json (hunk 1 above)
Comment (Lighteros) [reply package.json]: Fixed it on latest commit

File: app/components/competition/view/ViewCompetition.tsx (hunk 3 above)
Comment (Lighteros) [reply app/components/competition/view/ViewCompetition.tsx]: Fixed on latest commit

File: app/components/NavBar.tsx (hunk 4 above)
Comment (Lighteros) [reply app/components/NavBar.tsx]: Could you give me some details?

**Comment by ismellike** [comment]
image
^opening menu when viewing competition is overlapped by competition status badge
image
^View competition buttons are not properly aligned
Just a couple more UI tweaks
Also, it looks like we're missing a key on a list somewhere on HomePage. 
Then finally, let's make sure this is linted before merge. `yarn lint` `yarn lint:unsafe` can help here then use `yarn format`

🔥🔥🔥🔥 UI looks so nice now though great stuff
I'll get the homepage's text updated

**Pull request merged**